components/tooltip/tooltip.vue

```diff
@@ -343,6 +352,13 @@ export default {
     }
   },
 
+  beforeCreate () {
+    if (this.$props.disabled != null && this.$props.show != null) {
+      console.warn('Tooltip: You cannot use both the disabled and show props at the same time.');
+      console.warn('The show prop will be ignored.');
+    }
+  },
```

Comment:
Could we move this to the `mounted` hook?
I cannot reproduce this warn on `beforeCreate` probably its not being reach.

Reply:
will do.